sourcegraph/learn
# feat: add troubleshooting tutorial for python OSError

## Type of change

New feature: added tutorial for python OSError troubleshooting

### What should this PR do?

Solves #249 

### Why are we making this change?

Adds the tutorial for python OSError troubleshooting

### What are the acceptance criteria? 


- The post can be viewed locally by running `npm run dev` and navigating to the post.
- Check that the post has correct spelling and grammar

### How should this PR be tested?

- Check that the post has correct spelling and grammar (English is not my first language).
- Check if the content is correct. Because I've chosen one example of many functions that can raise this exception (In order to make it simple). Do I need to add more examples?

## Pull request process

**Reviewers**:

1. Test functionality using the criteria above.
2. Offer tips for efficiency, feedback on best practices, and possible alternative approaches and things that may not have been considered.
3. For shorter, "quick" PRs, use your best judgement on #​2.
4. Use a collaborative approach and provide resources and/or context where appropriate.
5. Provide screenshots/grabs where appropriate to show findings during review.

**Reviewees**:

1. Prefer incremental and appropriately-scoped changes.
2. Leave a comment on things you want explicit feedback on.
3. Respond clearly to comments and questions.
